Sources/FioriARKit/ARCards/Views/Components/AttachmentsView.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  AttachmentsView.swift
+//
+//
+//  Created by O'Brien, Patrick on 9/16/21.
+//
+
+import SwiftUI
+
+public struct AttachmentsItemModel: Identifiable {
```

Comment:
Proposal: what about `AttachmentUIMetadata` instead of `AttachmentsItemModel`

Reply:
Agreed, much better.